jekyll/_docs/introduction.md

```diff
@@ -29,17 +29,19 @@ If your tests pass, then you can deploy your code to development, staging, produ
 * [Heroku][doc-heroku]
 * [Deploy Using SSH][doc-sshdeploy]
 
-## How does CircleCI help you 
+## CircleCI's Role
 
-Now that we have a basic context of what CI and CD is, we can talk about how CircleCI fits into this picture. CircleCI integrates with your version control system (either GitHub or Bitbucket) and automatically runs a series of steps every time that we detect a change to your repository (when you push commits or a PR is opened). 
+Now that we have some context on CI and CD, we can discuss how CircleCI fits into this flow.
+
+CircleCI integrates with your version control system (GitHub/Bitbucket) and automatically runs a series of steps every time a change is detected in the repository (i.e. when you push commits or open a PR).
```

Comment:
Are there any other changes that we might run a build off of? (The difference between `i.e.` and `e.g.` is a pet issue.)